app/helpers/course/assessment/submission/submissions_autograded_helper.rb

```diff
@@ -20,7 +20,8 @@ def next_unanswered_question
   # The step that current user is on.
   def current_step
     @current_step ||= begin
-      @assessment.questions.index(@current_question) + 1
+      idx = @assessment.questions.index(@current_question)
```

Comment:
Can shorten this to `@current_question ? @assessment.questions.index(@current_question) + 1 : nil`